src/EnableNewWebImplementation.ts

```diff
@@ -3,15 +3,28 @@ import { Platform } from 'react-native';
 let useNewWebImplementation = true;
 let getWasCalled = false;
 
+/**
+ * @deprecated new web implementation is enabled by default. This function will be removed in Gesture Handler 3
+ */
 export function enableExperimentalWebImplementation(
   _shouldEnable = true
 ): void {
   // NO-OP since the new implementation is now the default
+  console.warn(
+    'New web implementation is enabled by default. This function will be removed in Gesture Handler 3'
+  );
 }
 
+/**
+ * @deprecated legacy implementation is no longer supported. This function will be removed in Gesture Handler 3
+ */
 export function enableLegacyWebImplementation(
   shouldUseLegacyImplementation = true
 ): void {
+  console.warn(
+    'Legacy web implementation is deprecated. This function will be removed in Gesture Handler 3'
```

Comment:
```suggestion
    'Legacy web implementation is deprecated. This function will be removed in Gesture Handler 3.'
```

Reply:
done in [0af4a93](https://github.com/software-mansion/react-native-gesture-handler/pull/3259/commits/0af4a93fac40dcc83d88587156a3db4c213a5e0b)